aws/importers.go

```diff
@@ -82,7 +82,7 @@ func Importers() map[string]core.Importer {
 		// "aws_snapshot_create_volume_permission": &AwsSnapshotCreateVolumePermissionImporter{},
 		// "aws_ebs_snapshot": &AwsEbsSnapshotImporter{},
 		"aws_ebs_volume": &AwsEbsVolumeImporter{},
-		// "aws_eip": &AwsEipImporter{},
+		"aws_eip": &AwsEipImporter{}, // TODO
```

Comment:
Expand TODO or remove and reference in the file instead?

Reply:
TODO was local as a flag of what I was working on; will remove